tests/distributions/test_stable.py

```diff
@@ -0,0 +1,57 @@
+import warnings
+
+import numpy as np
+import pytest
+import torch
+from scipy.integrate.quadpack import IntegrationWarning
+from scipy.stats import kstest, levy_stable
+
+import pyro.distributions as dist
+
+
+@pytest.mark.parametrize("sample_shape", [(), (7,), (6, 5)])
+@pytest.mark.parametrize("batch_shape", [(), (4,), (3, 2)])
+def test_shape(sample_shape, batch_shape):
+    stability = torch.empty(batch_shape).uniform_(0, 2).requires_grad_()
+    skew = torch.empty(batch_shape).uniform_(-1, 1).requires_grad_()
+    scale = torch.randn(batch_shape).exp().requires_grad_()
+    loc = torch.randn(batch_shape).requires_grad_()
+
+    d = dist.Stable(stability, skew, scale, loc)
+    assert d.batch_shape == batch_shape
+
+    x = d.rsample(sample_shape)
+    assert x.shape == sample_shape + batch_shape
+
+    x.sum().backward()
+
+
+@pytest.mark.parametrize("beta", [-0.6, -0.3, 0.0, 0.3, 0.6])
+@pytest.mark.parametrize("alpha", [0.2, 0.5, 0.8, 0.99, 1.0, 1.01, 1.2, 1.5, 1.8])
```

Comment:
is it stable for cases around boundaries of `alpha` and `beta`, e.g. `alpha=0.01 or 2`, `beta=-1 or 1`?

Reply:
the scipy reference implementation seems to break down in edge cases.

Reply:
Oh, I have added more edge cases! Note that scipy breaks down at `alpha=0.995` and `alpha=1.005`, so I have instead temporarily increased `RADIUS` so that interpolation logic is checked.

Reply:
Thanks for explaining!